tests/test_main.py

```diff
@@ -1 +1,56 @@
-# write your code here
+from unittest import mock
+from app.main import cryptocurrency_action
+
+
+@mock.patch("app.main.get_exchange_rate_prediction")
+def test_prediction_equal_current(prediction):
+    prediction.return_value = 10
+    assert cryptocurrency_action(10) == "Do nothing"
```

Comment:
well, this test failed because in this case cryptocurrency_action returns "Sell all your cryptocurrency"